src/main/resources/application.yml

```diff
@@ -198,7 +198,7 @@ thread-pool:
 
 logging:
   pattern:
-    level: "%X{correlationId}%5p"
+    level: "%X{correlationId} %5p"
```

Comment:
Minor correction to the logging pattern. Without the space it means that there is no space between the correlationID and the logger level when the logger level is 5 characters wide (such as `ERROR`)